Update Documentation in Conformance to the Documentation Guide

vishnuravi opened the pull request:
# Update Documentation in Conformance to the Documentation Guide

## :gear: Release Notes 
- Creates DocC documentation for setting up and using the module
- Updates the README to match the DocC documentation

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md):
- [X] I agree to follow the [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md).

vishnuravi committed:
Update Documentation in Conformance to the Documentation Guide

vishnuravi committed:
Crop images

vishnuravi committed:
Resize images

vishnuravi committed:
Improve README [skip ci]

vishnuravi committed:
Update Spezi Setup link

PSchmiedmayer committed:
Update README.md

PSchmiedmayer committed:
Update SpeziAccessGuard.md

PSchmiedmayer reviewed (changes requested):
Thank you for the great additions there!

As noted in the Storage and other Spezi PRs, it would be great if we can use this chance to also add the link checking PR here, we might have similar issues in this repo and it would be great to automatically check this. Really like the new GitHub Action, this is very useful and already caught a few issues for me.

Would be great if we can also add a schedule here 👍

vishnuravi committed:
Add markdown link check (#15)

# Add markdown link check

## :gear: Release Notes 
Adds an action to check markdown files in the repository for broken
links, both on pull requests, and also at midnight on the first day of
every month.

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code
of
Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md)
and [Contributing
Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md):
- [X] I agree to follow the [Code of
Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md)
and [Contributing
Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md).

PSchmiedmayer reviewed (approved):
Thank you, looks great @vishnuravi! 🚀
